src/components/data-table/PDataTableHeader.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <th @click="$emit('click')">
+    <slot/>
+    <template v-if="sort">
+      <template v-if="$slots.sort">
```

Comment:
Is this being used?

Reply:
I'll implement its usage on PDataTable and add stories to cover the use cases